homeassistant/components/notify/__init__.py

```diff
@@ -132,6 +151,10 @@
             SERVICE_NOTIFY)
         platform_name_slug = slugify(platform_name)
 
+        if getattr(notify_service, 'can_dismiss', False):
```

Comment:
What services support this? Do we have a precedence for optionally registering a base component service depending on platform support?

Reply:
i've only implemented this in html5. I have not looked at all the other services to see which could support it.
I thought of moving html5 out to its own platform but that would mean breaking change i think.

Reply:
The `targets` attribute that optionally registers a notify service gives some precedence. So this is in line with that. But is this they way we should extend services? Do we think it's scalable? I'm not saying it isn't. But I'm not sure it is either. So would be good with some comments and views on this.